arch/arm/src/armv7-m/mpu.h

```diff
@@ -33,8 +33,6 @@
 #  include <stdbool.h>
 #  include <assert.h>
 #  include <debug.h>
-
-#  include "arm_arch.h"
```

Comment:
Do we need to change to `arm_internal.h`?

Reply:
Don't, since mpu.h doesn't use putregx/getregx/modifyregx.